tests/Integration/Common/SpanAssertionTrait.php

```diff
@@ -16,4 +17,26 @@ public function assertExpectedSpans($testCase, $traces, $expectedSpans)
     {
         (new SpanChecker($testCase))->assertSpans($traces, $expectedSpans);
     }
+
+    /**
+     * Checks that the provide span exists in the provided traces and matches expectations.
+     *
+     * @param TestCase $testCase
+     * @param $traces
```

Comment:
Would this be the same here too?
```suggestion
     * @param Span[][] $traces
```